config.py

```diff
@@ -1147,16 +1210,18 @@ def get_picks(info):
     """Return the names of the channels we wish to analyze.
     """
     if get_kind() == 'meg' and ('mag' in ch_types or 'grad' in ch_types):
-        pick_idx = mne.pick_types(info, eog=True, ecg=True)
+        pick_idx = mne.pick_types(info, eog=True, ecg=True, exclude=[])
```

Comment:
@hoechenberger why did you have to do this?

Reply:
This functions is intended to return the subset of the channels that match the recording kind & channel type that the user wishes to analyze. It is called in the very first processing step. We want to ensure we include ALL channels here, including bads. I have renmed the functions and added more comments for clarification in c281eeb